internal/worker/toolexec_test.go

```
@@ -1167,3 +1171,187 @@ func TestCoordinatorSkipsDelegationCalls(t *testing.T) {
+	var fired atomic.Bool
+	var answer domain.ID
+	// The test goroutine reports, because t.Fatalf from a handler is
+	// runtime.Goexit. Buffered so the handler never blocks on the send.
+	injected := make(chan error, 1)
+	h = newHarnessWrapped(t, sb, func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			// Before forwarding, so the control plane sees the answer already
+			// on the log when it validates the result this request carries.
+			if r.Method == http.MethodPost && strings.HasSuffix(r.URL.Path, "/events") &&
+				fired.CompareAndSwap(false, true) {
+				injected <- h.answerAsPlatformErr(answer)
```

golangci-lint (2.12.2)

[error] Function `answerAsPlatformErr` should pass the context parameter

(contextcheck)